tomb

```diff
@@ -110,16 +110,20 @@ export TEXTDOMAIN=tomb
 
 # {{{ Safety functions
 
-# Wrap sudo with a more visible message
+# Wrap sudo with a more visible message or apply user-supplied alternative to sudo
 _sudo() {
-	local msg="[sudo] Enter password for user ::1 user:: to gain superuser privileges"
-	command -v gettext 1>/dev/null 2>/dev/null && msg="$(gettext -s "$msg")"
-	msg=${(S)msg//::1*::/$USER}
-	sudo -p "
+	if option_is_set --sudo; then
+		$(option_value --sudo) ${@};
```

Comment:
Better parse only valid values here in a `case/esac` list to avoid execution injections of sorts.

Reply:
Will do, but if I'm not mistaken, nothing is stopping an attacker from calling their script **doas** and doing something malicious under the hood. So this may lead to a false sense of security.

Reply:
It is just one small step towards better security. There are solutions for the issue you raise but they are beyond the scope of this change I guess.